stopcovid/dialog/engine.py

```diff
@@ -244,10 +244,16 @@ def _check_response(
                     )
                 )
                 if dialog_state.is_next_prompt_last():
+                    auto_continue = False
+                    current_drill = dialog_state.current_drill
+                    if isinstance(current_drill, Drill) and current_drill.auto_continue:
```

Comment:
Are we checking here to make sure that the `current_drill` is not None? If so, we could simplify this to be `auto_continue=current_drill and current_drill.auto_continue`

Reply:
I did this because when I looked at user state in DDB I noticed that when current_drill is not a drill it's value is True not something falsy. I think I have something cleaner that works.